app/server/integration/account-resolvers.test.ts

```diff
@@ -0,0 +1,75 @@
+import bcrypt from 'bcryptjs';
+import { getOrCreateServer } from '../src/core/server';
+import { getPrismaClient } from '../src/core/utils';
+import { createMercuriusTestClient } from 'mercurius-integration-testing';
+import * as plugins from '../src/core/plugins';
+import { redisEmitter } from '../src/core/plugins/mercurius';
+import * as jwt from '../src/lib/jwt';
+import faker from '@faker-js/faker';
+import { toGlobalId } from '../src/features/utils';
+
+const server = getOrCreateServer();
+const testClient = createMercuriusTestClient(server);
+const prisma = getPrismaClient(server.log);
+
+const toUserId = toGlobalId('User');
+const userData = {
+    email: faker.internet.email(),
+    firstName: faker.name.firstName(),
+    lastName: faker.name.lastName(),
+    id: faker.datatype.uuid(),
+    password: faker.internet.password(),
```

Comment:
This is a different open source one. Not maintained by the same guy who protested. [faker repo](https://github.com/faker-js/faker)

Reply:
Hmm, I see.  As I'm thinking about this more, I don't think we should be using faker for tests because it could lead to test flakiness.  Consider a scenario where we have a simple function such as `isPasswordStrongEnough`.  We could have a scenario where tests intermittently fail due to faker generating a password that is strong enough 99% of the time, but occasionally fails, which would be very hard to debug.  Now take this scenario and apply it to hundreds of tests throughout the codebase.

I know I used faker for tests in the past, but now I think that's a mistake.  Especially from a reliability perspective.

Reply:
Alright if you think it's better to just hardcode the values I can do that.